src/bin/svg2png/svg2png.cpp

```diff
@@ -198,12 +198,11 @@ struct App
                 } else if (p[1] == 'b') {
                     //image background color
                     if (!p_arg) {
-                        cout << "Error: Missing background color attribute. Expected eg. -b #fa7410." << endl;
+                        cout << "Error: Missing background color attribute. Expected eg. -b fa7410." << endl;
                         return 1;
                     }
 
-                    if (*p_arg == '#') ++p_arg;
```

Comment:
I prefer to leave it. The rest is ok.

Reply:
do you have some arguments for that? 
bash interprets the # sign in a  given way and its not the way we expect here